website/docs/r/repository_topics.html.markdown

```diff
@@ -0,0 +1,43 @@
+---
```

Comment:
Do you mind linking to this doc in the `github.erb` file so it's discoverable?

Reply:
Done! Sorry for missing that!